packages/graphql/schema.graphql

```diff
@@ -3022,6 +3022,8 @@ type JoinGroupResponse {
 
 union JoinGroupResult = JoinGroupResponse | GroupOperationValidationFailed | SponsoredTransactionRequest | SelfFundedTransactionRequest | TransactionWillFail
 
+scalar JsonString
```

Comment:
Can we defined this under `packages/types/src/misc.ts` as a branded string, and map it in the types scalars?